extensions/mini/__init__.py

```diff
@@ -84,7 +89,7 @@ def register(self):
 
     async def color(self, args, message):
         parser = argparse.ArgumentParser(description=self.bot.processOutput("Obtain a colo(u)r from a role on the server", message))
-        parser.add_argument("role", help=self.bot.processOutput("The role to get the color of", message))
+        parser.add_argument("role", nargs="?", default=str(message.author.id), help=self.bot.processOutput("The role to get the color of", message))

```

Comment:
This won't work, we need to get the role id, but you are defaulting to the authors id